### PR TITLE
Add SC tech intro

### PR DESCRIPTION
**Pull request by ppeble**

### CODE REVIEW

File: _posts/2015-04-02-welcome-to-sittercity-tech.markdown [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Welcome to Sittercity Technology!"
+date:   2015-04-02 09:00:00
+categories: sittercity
+---
+
+Welcome! This here is our new blog! As is fitting of a new journal let's start ourselves off with an introduction to Sittercity Tech.
```
Comment (jazminschroeder) [inline _posts/2015-04-02-welcome-to-sittercity-tech.markdown]: Reword opening. `This here`

Comment (ppeble) [inline _posts/2015-04-02-welcome-to-sittercity-tech.markdown]: I did that intentionally to be all folksy. :smile: 

I can take it out if you think it just looks like an error. I'm not attached to it.

File: _posts/2015-04-02-welcome-to-sittercity-tech.markdown [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+####  What we do
+
+So! [Sittercity](https://www.sittercity.com). What do we do? Sittercity is a service that connects families and sitters. Our goal is to
```
Comment (jazminschroeder) [inline _posts/2015-04-02-welcome-to-sittercity-tech.markdown]: I think we need to get help from marketing for this first paragraph. We need to make sure that what we say here(What we do) aligns with the message that the company is putting out there.

Comment (ppeble) [inline _posts/2015-04-02-welcome-to-sittercity-tech.markdown]: That's cool, I'm open to that.

Comment (ppeble) [inline _posts/2015-04-02-welcome-to-sittercity-tech.markdown]: I'll reach out to someone from Marketing on Monday.

File: _posts/2015-04-02-welcome-to-sittercity-tech.markdown (hunk 1 above)
Comment (jazminschroeder) [inline _posts/2015-04-02-welcome-to-sittercity-tech.markdown]: Aaaaah  I didn't get it but let's see what other's say! I'm still learning this folksy stuff

**Comment by jazminschroeder** [comment]
Awesome. A couple of thoughts

**Pull request merged**